### DISCUSSION TITLE
new screen width issue on grid

### DISCUSSION THREAD

**Discussion by BhavikTrambadiya** [tailwindlabs/tailwindcss#8035] [Help]
#### I am trying to use the grid at the sales banner at the home but when I use screen width media query on that but not apply on the grid. so I don't know the right solution or that is a bug in this issue.

The main problem at the 
![image1](https://user-images.githubusercontent.com/77911712/161435837-a08e0fde-5d31-4760-8735-ce0a2c5c0dce.png)
![image2](https://user-images.githubusercontent.com/77911712/161435850-7c779335-11e4-4672-9c71-d72149d1018e.png)
![image3](https://user-images.githubusercontent.com/77911712/161435852-aa1d6acd-69bd-4f54-89aa-12d1c037eefa.png)

### HTML in text file
[query.txt](https://github.com/tailwindlabs/tailwindcss/files/8404897/query.txt)

### tailwind CSS file

>   .flex-all-center {
        @apply flex items-center justify-center;
    }

    .flex-all-between {
        @apply flex items-center justify-between;
    }

### Tailwind config

> module.exports = {

    content: [
        "./resources/**/*.blade.php",
        "./resources/**/*.js",
        "./resources/**/*.vue",
    ],
    theme: {
        extend: {
            screens: {
                xs: "476px",
            },
            colors: {
                darkgray: "#666666",
                lightgray: "#F1F1F1",
                themegray: "#C4C4C4",
                themeblue: "#0062F5",
                black: "#212121",
                ghostwhite: "#FBFCFF",
            },
        },
    },
    plugins: [],
};

## The problem at this  point
![Screenshot 2022-04-03 205107](https://user-images.githubusercontent.com/77911712/161435855-e2704262-c846-4a70-9e06-f505232487b7.png)
## And I want this
![Screenshot 2022-04-03 205148](https://user-images.githubusercontent.com/77911712/161435856-e8defe8d-ed72-4bb3-afb5-b05ea9a26161.png)

if you want that image at the [tailwind play](https://play.tailwindcss.com/) 
![banner1](https://user-images.githubusercontent.com/77911712/161436346-d9e6b85d-a1e6-4c3b-8892-a9e07255de58.png)
![banner2](https://user-images.githubusercontent.com/77911712/161436348-690e86a8-f4e7-4133-842e-4d2bb3ca8ae6.png)
![banner3](https://user-images.githubusercontent.com/77911712/161436349-52bea982-260b-4227-a00c-c2b4f3eeac82.png)

**Comment by CFarhad** [comment]
I can answer your question but give me a gif or video about that. I'm confused
